main/python/cmdLineUtils.py

```diff
@@ -641,11 +641,11 @@ def copyRootObjectRecursive(sourceFile,sourcePathSplit,destFile,destPathSplit,re
                         retcode += retcodeTemp
                         continue
                     else:
-                        obj.SetName(setName)
+                        if obj.InheritsFrom(ROOT.TNamed.Class()): obj.SetName(setName)
```

Comment:
A more pythonic way to check this would be `if isinstance(obj, ROOT.TNamed)`, `isinstance` also checks subclasses.